docs/tasks/20260425_1606_sankey-svg検索フィルタ機能設計.md

```
@@ -0,0 +1,157 @@
+- **事業**: 支出額下限 / 支出額上限（project-spending の value）
+- **支出先**: 支出額下限 / 支出額上限（recipient の value）
+
+単位: 円（UIでは「億円」表示、内部計算は1円単位のまま）
+
+---
+
+## アーキテクチャ
+
+### レイヤー変更範囲
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)